docusaurus/docs/reactnative/ui-components/auto_complete_input.mdx

```diff
@@ -26,7 +26,7 @@ Component to render input box within within the [`MessageInput`](./message_input
 This component provides autocomplete functionality for the commands and mentions feature.
 Commands suggestions will show up when user types "`/`", while mentions suggestions will show up when user types "`@`".
 
-Please check the guide of [Customize Message Input](../guides/message_input_customization.mdx) for usage of this component.
+Check the [Customize Message Input guide]([Customize Message Input](../guides/message_input_customization.mdx)) for more details about how to customize this component
```

Comment:
This one is my bad but I think `more details on how to...` feels better than `more details about how to...`. My mistake in the review tho. You can ignore this comment.